lib/connect.js

```diff
@@ -375,8 +393,38 @@ const LogLevel = {
   off: 'off'
 };
 
-function createLocalParticipant(signaling, log, encodingParameters, options, localTracks) {
-  const localParticipantSignaling = signaling.createLocalParticipantSignaling(encodingParameters);
+/**
+ * The verbosity level of network quality information of a {@link Participant}.
+ * @enum {number}
+ */
+// eslint-disable-next-line
+const NetworkQualityVerbosity = {
+  /**
+   * Nothing is reported for the {@link Participant}. This has no effect and
+   * defaults to {@link NetworkQualityVerbosity#minimal} for the {@link LocalParticipant}.
+   */
+  none: 0,
+  /**
+   * Reports {@link NetworkQualityLevel} for the {@link Participant}.
+   */
+  minimal: 1,
+  /**
+   * Reports {@link NetworkQualityLevel} and {@link NetworkQualityStats} for the {@link Participant}.
+   * {@link NetworkQualityStats} is populated with audio and video {@link NetworkQualityLevel}s
+   * based on which the {@link Participant}'s {@link NetworkQualityLevel} is calculated.
```

Comment:
Do we need to mention "based on which the {@link Participant}'s {@link NetworkQualityLevel} is calculated"?
Ive seen that in a few places. What's the purpose?

Reply:
@aymenn ,

Yes. We need to clarify that the stats are the basis for our calculation of the NQ level.

Reply:
Thanks @manjeshbhargav. I really meant more, why do we need to clarify it? and why do we need to clarify it so may times?

Reply:
@aymenn ,

The idea is that there are different paths that customers can use to navigate the docs, and having these kinds of clarifications at multiple places makes sure that nobody misses them.
Does that make sense?

Reply:
Ok I think in the future we dont need to. This for example should just be mentioned where we document networkQualityLevel. Mentioning at every occurrence adds little value. i.e. less is more.